pkgs/development/python-modules/ingredient-parser-nlp/default.nix

```
@@ -0,0 +1,56 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  nix-update-script,
+
+  setuptools,
+
+  nltk,
+  python-crfsuite,
+  pint,
+  floret,
+
+  pytestCheckHook,
+  nltk-data,
+}:
+buildPythonPackage rec {
+  pname = "ingredient-parser-nlp";
+  version = "2.0.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "strangetom";
+    repo = "ingredient-parser";
+    tag = version;
+    hash = "sha256-i14RKBcvU56pDNGxNVBvvpQ65FCbitMIfvN5eLLJCWU=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    nltk
+    python-crfsuite
+    pint
+    floret
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+  ];
+
+  pythonImportsCheck = [
+    "ingredient_parser"
+  ];
+
+  # Needed for tests
+  env.NLTK_DATA = nltk-data.averaged_perceptron_tagger_eng;
```

Nit: if it's only needed for tests, it probably should be set in `preCheck`:

```suggestion
  preCheck = ''
    export NLTK_DATA=${nltk-data.averaged_perceptron_tagger_eng}
  '';